client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -276,38 +277,35 @@ class BusinessDetails extends Component {
 		const revenueOptions = [
 			{
 				key: 'none',
-				label: _x( "$0 (I'm just getting started)", '$0 revenue amount', 'woocommerce-admin' ),
+				label: sprintf(
+					_x( "%s (I'm just getting started)", '$0 revenue amount', 'woocommerce-admin' ),
+					formatCurrency( 0 )
+				),
 			},
 			{
 				key: 'up-to-2500',
-				label: _x( 'Up to $2500 USD', 'equivalent of $2500 USD', 'woocommerce-admin' ),
+				label: sprintf(
+					_x( 'Up to %s', 'Up to a certain revenue amount', 'woocommerce-admin' ),
```

Comment:
Missing `/* Translators....` I also wonder if `_x` is needed after adding translators comments for this and the other occurrences in this PR. Context is used to distinguish strings with different meaning but the same content - why do these need special context?

Reply:
Added in https://github.com/woocommerce/woocommerce-admin/pull/3324/commits/9a88072b8c8265120be44eab1af7d9b26bb1b064.

I think the context may have been a previous iteration when formatting for revenues and product count were handled together.  Removed as it does not appear to be necessary currently.